classes/blocks/class-carouselheader.php

```diff
@@ -22,17 +22,29 @@ class CarouselHeader extends Base_Block {
 	 *
 	 * @const string BLOCK_NAME.
 	 */
-	const BLOCK_NAME = 'carousel-header-beta';
+	const BLOCK_NAME = 'carousel-header';
 
 	/**
-	 * Gallery constructor.
+	 * CarouselHeader constructor.
 	 */
 	public function __construct() {
+		add_action( 'init', [ $this, 'register_carouselheader_block' ] );
+	}
+
+	/**
+	 * Register CarouselHeader block.
+	 */
+	public function register_carouselheader_block() {
+		wp_enqueue_script( 'hammer', 'https://cdnjs.cloudflare.com/ajax/libs/hammer.js/2.0.8/hammer.min.js', [], '2.0.8', true );
```

Comment:
This enqueue was previously in `enqueue_editor_scripts` function. If we add it here, it will be added to each page on the front end.